docs/pages/api/og.tsx

```diff
@@ -69,16 +69,22 @@ export default async function openGraphImage(
 ): Promise<ImageResponse> {
   try {
     const [fonts, bg] = await loadAssets();
-    const { searchParams } = new URL(req.url!);
+    // @ts-expect-error -- Shouldn't a request always have a url?
+    const { searchParams } = new URL(req.url);
```

Comment:
Looks like `NextApiRequest` extends `IncomingRequest` and the type comes from here: 

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e2988a134ec9fa51deb43487231ac607265ba088/node/index.d.ts#L708-L711